src/main/java/org/nextoperation/memory/repository/MultiValueRepository.java

```diff
@@ -0,0 +1,58 @@
+package org.nextoperation.memory.repository;
+
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Extended repository with multi-value support.
+ *
+ * @since 1.1.0
+ * @author haril song
```

Comment:
```suggestion
 * @author haril song
```
@khy07181 해당 기능의 아이디어를 주셨는데 원하는 이름으로 author 추가해주세요~ 아직 초안 수준이라 더 만들어야하는 것도 많은데 필요하신거 제안해주시거나 기여해주시면 더욱 좋겠네요 👍